src/main/java/org/autopotato/linkedinclonebackend/controllers/ConnectionController.java

```diff
@@ -0,0 +1,86 @@
+package org.autopotato.linkedinclonebackend.controllers;
+
+import java.util.NoSuchElementException;
+import javax.validation.constraints.NotNull;
+import lombok.Value;
+import org.autopotato.linkedinclonebackend.model.ConnectionRequest;
+import org.autopotato.linkedinclonebackend.services.ConnectionRequestService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.validation.annotation.Validated;
+import org.springframework.web.bind.annotation.*;
+
+@RestController
+@RequestMapping("connections")
+public class ConnectionController {
+    private final ConnectionRequestService connectionRequestService;
+
+    @Autowired
+    public ConnectionController(ConnectionRequestService connectionRequestService) {
+        this.connectionRequestService = connectionRequestService;
+    }
+
+    /**
+     * JSON Mapping class for Simple response JSON
+     */
+    @Value
+    public static class SimpleResponseDTO {
+        String message;
+    }
+
+    /**
+     * JSON Mapping class for JSONs involved with creating a new Connection Request
+     */
+    @Value
+    public static class NewConnectionRequestDTO {
+        @NotNull
+        Long senderId;
```

Comment:
We can/should also check for minimum value using something like `Min(1)`.

Reply:
I have added the annotations

Reply:
We could also write tests for negative values and for 0.

Reply:
I already did